lib/gourami/coercer.rb

```diff
@@ -23,12 +21,17 @@ def setter_filter(attribute_name, value, options)
     #
     # @param value [Object]
     # @option allow_nil [Boolean] (true)
+    # @option nil_when_empty [Boolean] (true)
     # @option upcase [Boolean] (false)
     # @option strip [Boolean] (true)
     #
     # @return [String, nil]
     def coerce_string(value, options = {})
-      return if value.nil? && options.fetch(:allow_nil, true)
+      if options.fetch(:allow_nil, true)
+        return if value.nil?
+        value = value.to_s
```

Comment:
is this necessary here ?

Reply:
Yes, it can receive any data type, such as `true`. true does not respond to `empty?`